central/notifiers/jira/jira.go

```diff
@@ -280,9 +280,6 @@ func newJira(notifier *storage.Notifier, metadataGetter notifiers.MetadataGetter
 	decCreds := notifier.GetJira().GetPassword()
 	var err error
 	if env.EncNotifierCreds.BooleanSetting() {
-		if notifier.GetNotifierSecret() == "" {
```

Comment:
According to `Validate` a password or token is required? @kylape is that true in the new model as well?

Reply:
Same as above

Reply:
`Validate` will _not_ fail if encrypted secrets is enabled.

See the 2nd param here: https://github.com/stackrox/stackrox/blob/68aea4d8ef00806b0036d55640b2ce83a33a07f0/central/notifiers/jira/jira.go#L276

Which is used to skip the check in validate: https://github.com/stackrox/stackrox/blob/68aea4d8ef00806b0036d55640b2ce83a33a07f0/central/notifiers/jira/jira.go#L251

Reply:
It looks like `notifier.Config.Password` is used when encryption is disabled, and `notifier.NotifierSecret` is used when encryption is enabled.  Thus I would expect `jira.GetPassword()` to be empty when credential encryption is enabled, right?  If so, then it looks like line 251 should be `if !validateSecret && jira.GetPassword() == "" {`.

Reply:
Going to take this offline since it is convoluted with multiple paths, to say the least. The tldr is the removal of the check in all notifiers, is ok.